### PR TITLE
test:fix flaky test

### PR DESCRIPTION
**Pull request by leizhiyuan**
### Motivation:

Explain the context, and why you're making that change.
To make others understand what is the problem you're trying to solve.

### Modification:

Describe the idea and modifications you've done.

### Result:

Fixes #<GitHub issue number>. 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

File: core/api/src/test/java/com/alipay/sofa/rpc/client/ProviderInfoTest.java [hunk 1]
```diff
@@ -42,8 +43,9 @@ public void test() {
 
     @Test
     public void testGetWeight() {
+        //1s
         ProviderInfo provider = ProviderHelper
-            .toProviderInfo("bolt://10.15.232.229:12222?timeout=3333&serialization=hessian2&connections=1&warmupTime=6&warmupWeight=5&appName=test-server&weight=2000");
```
Comment (leizhiyuan) [inline core/api/src/test/java/com/alipay/sofa/rpc/client/ProviderInfoTest.java:46]: 6ms，ci机器不行，很容易ci挂

File: metrics/metrics-lookout/src/main/java/com/alipay/sofa/rpc/metrics/lookout/RpcLookout.java [hunk 2]
```diff
@@ -220,7 +220,7 @@ private Id createMethodConsumerId(RpcClientLookoutModel model) {
         tags.put("invoke_type", StringUtils.defaultString(model.getInvokeType()));
         tags.put("target_app", StringUtils.defaultString(model.getTargetApp()));
 
-        return rpcLookoutId.fetchConsumerStatId().withTags(tags);
+        return rpcLookoutId.fetchConsumerStatId(tags);
```
Comment (leizhiyuan) [inline metrics/metrics-lookout/src/main/java/com/alipay/sofa/rpc/metrics/lookout/RpcLookout.java:223]: 修改为直接创建，避免歧义，并且里面可以存cache

**Review by OrezzerO — APPROVED** [review]
LGTM

**Review by EvenLjj — APPROVED** [review]
LGTM

**Pull request merged**